splunk-conf.YAML-tmLanguage

```diff
@@ -13,7 +13,7 @@ patterns:
   match: ^# DO NOT EDIT THIS FILE\!$
   name: invalid.illegal
 - comment: A comment
-  match: ^#.*$
+  match: ^[#|@].*$
```

Comment:
I don't think this is the right regex. It should either be `^[#@].*$` or `^(#|@).*$`, I'd prefer the former